src/test/resources/application-local.yaml

```diff
@@ -1,10 +1,8 @@
 spring:
-  redis:
-    host: 127.0.0.1
-    password: foobared # default passord lokalt i /usr/local/etc/redis.conf
-  cache:
+  data:
     redis:
-      time-to-live: 30000 # 30 sek i millis
+      ssl: false
+      password: leeloodallasmultipass
```

Comment:
For å fremprovosere feilen jeg fikk ved deploy til dev-gcp så må passord enables på redis som brukes i testene. Dette er også lagt til i gh workflow. 
Skrev noen ord om hva som er årsaken til feilen sammen med workaround/fiksen i RedisConfig.kt